src/print.jl

```diff
@@ -634,13 +651,32 @@ function constraint_string(
         return prefix * constraint_str
     end
 end
+
+"""
+    constraint_string(
+        print_mode,
+        ref::ConstraintRef;
+        in_math_mode::Bool = false)
+
+Return a string representation of the constraint `ref`, given the `print_mode`.
+
+`print_mode` should be `IJuliaMode` or `REPLMode`.
+"""
 function constraint_string(print_mode, ref::ConstraintRef; in_math_mode = false)
     return constraint_string(print_mode, name(ref), constraint_object(ref), in_math_mode = in_math_mode)
 end
 
 #------------------------------------------------------------------------
 ## _NonlinearExprData
 #------------------------------------------------------------------------
+
+"""
+    nl_expr_string(model::Model, print_mode, c::_NonlinearExprData)
+
+Return a string representation of the nonlinear expression `c` belonging to
+`model`, given the `print_mode`. `print_mode` should be `IJuliaMode` or
+`REPLMode`.
+"""
 function nl_expr_string(model::Model, print_mode, c::_NonlinearExprData)
```

Comment:
Ditto on whether this should be exported.

Reply:
We should probably revisit these questions once https://github.com/jump-dev/MathOptInterface.jl/pull/1180 is done. I'm waitinh

Reply:
The printing PR is ready from my side: https://github.com/jump-dev/JuMP.jl/pull/2449